full-stack-tests/presentation/package.json

```diff
@@ -44,7 +44,6 @@
     "@types/chai-as-promised": "^7",
     "@types/chai-jest-snapshot": "^1.3.0",
     "@types/chai-subset": "1.3.1",
-    "@types/cpx": "1.5",
```

Comment:
I think you may still need this dep

Reply:
yeah, I just wanted to reduce the noise with the first commit so I could see which changelog files were still relevant. I got the rest in separate commits